client/src/pages/fundraisers/FundraiserCreate.js

```diff
@@ -14,7 +13,6 @@ import {
 } from "@material-ui/core";
 
 import { withStyles } from "@material-ui/core/styles";
-import Snackbar from "@material-ui/core/Snackbar";
```

Comment:
Please add the snackbar validation to render where necessary, you can replicate the implementation from the signin/signup pages